modules/network/src/main/java/org/apache/ignite/network/scalecube/ScaleCubeClusterServiceFactory.java

```diff
@@ -51,7 +52,7 @@
                     topologyService.onMembershipEvent(event);
                 }
             })
-            .config(opts -> opts.memberAlias(context.getName()))
+            .config(opts -> opts.memberAlias(context.getName()).membership(config -> config.suspicionMult(1)))
```

Comment:
please add a comment that this timeout value is currently needed for tests

Reply:
also, please move the multiplier configuration to the `membership` configuration on line 57

Reply:
Why tho? TODO in class def already states that it would be changed

Reply:
yeah, maybe it's not really needed after all =)